fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/lao/federation/LinkedOrganizationsInviteFragment.kt

```diff
@@ -27,48 +35,93 @@ class LinkedOrganizationsInviteFragment : Fragment() {
   @Inject lateinit var networkManager: GlobalNetworkManager
 
   private lateinit var laoViewModel: LaoViewModel
+  private lateinit var linkedOrganizationsViewModel: LinkedOrganizationsViewModel
 
   override fun onCreateView(
       inflater: LayoutInflater,
       container: ViewGroup?,
-      savedInstanceState: Bundle?
-  ): View? {
+      savedInstanceState: Bundle?,
+  ): View {
     val binding = LinkedOrganizationsInviteFragmentBinding.inflate(inflater, container, false)
     laoViewModel = LaoActivity.obtainViewModel(requireActivity())
+    linkedOrganizationsViewModel =
+        obtainLinkedOrganizationsViewModel(requireActivity(), laoViewModel.laoId)
 
-    binding.linkedOrganizationsServerText.text = networkManager.currentUrl
+    if (CREATES_INVITATION) {
+      // When the user creates the invitation
```

Comment:
I guess a previous PR got blindly merge as I don't remember reviewing this. Why is `CREATES_INVITATION `defined as static constant when then it's not? You could have simply passed it in a custom fragment constructor